### ISSUE TITLE
Framed Drawers bug when facing South

### ISSUE THREAD

**Issue by aaronhowser1** [jaquadro/StorageDrawers#686]
This is before I place something into it
![image](https://user-images.githubusercontent.com/15698722/50096815-2cf2fc00-01de-11e9-810c-4fd0e862a554.png)
It turns to face East after I place things it. It doesn't show the item, and I can only insert/extract from the South side still
![image](https://user-images.githubusercontent.com/15698722/50096919-5b70d700-01de-11e9-8bb1-5361e0802490.png)
I've tested all other directions and they all work normally, and normal Drawers work facing South too.

**Comment by RobertLSnead** [comment]
I'm having the same issue also!
It appears the face wants to rotate to another direction, the items are still in the unit, you just can not see them.

**Comment by Shadowhunter82** [comment]
I have the same issue with the normal drawers,
In my opinion its no Storage Drawers problem --> on my server it only happens if there is a Block update nearby or if i try to add items to the drawer (rightklick), on the leftklick the drawer works fine. could be related to another mod??
And sometimes it works normal to (could be related to mod loading order to)

**Comment by RobertLSnead** [comment]
It's an issue for sure.  The block literately rotates away from the player when placing it down facing north.
Also when you place an item in the drawer, the drawer on all sides default to the base texture.  This happens on my server whether or not a tick update happens or not!

**Comment by RobertLSnead** [comment]
It does not happen on single player game play,  only on the multiplayer server side.
Sounds like a packet issue.

**Comment by Shadowhunter82** [comment]
I don´t think so .... on my server / single player test it sometimes works and sometimes not, in some situations if i change the forge version it works 1-2 times and than the same issue comes back

**Comment by RobertLSnead** [comment]
Well I can tell you it is doing it 100% for me...  I'm playing on GregBlock apparently an older forge version.
Which forge version seemed to work the best for you?

**Comment by RobertLSnead** [comment]
This happens also on StoneBlock2 mod pack!

**Comment by Delfite** [comment]
I'm getting this issue as well on my own PERSONAL modpack I'm making, this could've been caused by three mods. Removing serene seasons, removing mobends, or adding moar tinkers.
I have ZERO IDEA if these mods are even close to the cause but I will try to narrow it down and post a temporary fix here!

**Comment by Delfite** [comment]
FOUND THE ISSUE! As a temporary fix for this, remove or disable MoarTinkers from your modpack if you have storage drawers and are getting this same problem. As soon as I removed the mod and loaded into my world all of my drawers reverted to normal.
"You're welcome."

**Comment by ShadowDrakken** [comment]
I don't have MoarTinkers and am still having this issue on the StoneBlock 2 pack :(
